app/controllers/games_controller.rb

```diff
@@ -11,17 +11,18 @@ def show
 
     def create 
         game = Game.create(game_params)
+        game.user_id = params[:game][:user_id] # Set the user_id explicitly
```

Comment:
Since the strong params have the user_id, question: what purpose is being served by line 14?

Reply:
that line does nothing so it can be removed. it doesn't hurt the code either though so that is why we missed it.